resources/views/platform/components/game/link-buttons/index.blade.php

```diff
@@ -0,0 +1,103 @@
+@props([
+    'allowedLinks' => ['forum-topic', 'game-files', 'guide', 'code-notes', 'tickets', 'set-requestors'],
+    'gameAchievementsCount' => 0,
+    'gameForumTopicId' => null, // ?int
+    'gameGuideUrl' => null, // ?string
+    'gameId' => 1,
+    'isViewingOfficial' => false,
+    'variant' => 'stacked', // 'stacked' | 'row'
+])
+
+<?php
+use App\Community\Enums\TicketFilters;
+use App\Community\Models\ForumTopic;
+use App\Site\Enums\Permissions;
+use Illuminate\Support\Facades\Auth;
+
+$me = Auth::user();
+
+$doesForumTopicExist = false;
+if ($gameForumTopicId) {
+    $doesForumTopicExist = ForumTopic::where('ID', $gameForumTopicId)->exists();
+}
+
+$canCreateForumTopic = !$doesForumTopicExist && $me && $me->Permissions >= Permissions::Developer;
+
+$canSeeForumLink = in_array('forum-topic', $allowedLinks);
+$canSeeSupportedGameFiles = in_array('game-files', $allowedLinks) && $me && $me->Permissions >= Permissions::Registered;
+$canSeeCodeNotes = in_array('code-notes', $allowedLinks) && $me && $me->Permissions >= Permissions::Registered;
+$canSeeGuide = in_array('guide', $allowedLinks) && $gameGuideUrl;
+$canSeeOpenTickets = in_array('tickets', $allowedLinks) && $me && $me->Permissions >= Permissions::Registered;
+$canSeeSetRequestors = in_array('set-requestors', $allowedLinks) && $me && $me->Permissions >= Permissions::Registered && $gameAchievementsCount === 0;
+
+if (in_array('tickets', $allowedLinks) && $canSeeOpenTickets) {
```

Comment:
`$canSeeOpenTickets` encapsulates `in_array('tickets, $allowedLinks)` now.

Reply:
Thanks - changed to just `if ($canSeeOpenTickets)`.